humanguide/design-concepts/app-icon-design.rst

```diff
@@ -0,0 +1,109 @@
+App icon design
+===============
+
+The whole system and apps follow the `Suru design principles <http://web.archive.org/web/20191223082720/https://docs.ubuntu.com/phone/en/apps/design/get-started/design-philosophy>`__ started by Canonical and `Samuel Hewitt <https://icons.samuelhewitt.com/>`__.
+
+Design Philosophy and Suru Meaning
+----------------------------------
+
+-  Suru alludes to Japanese culture.
+-  The design of Suru is inspired by origami.
+-  Paper can be used in two and three dimensional forms, because it is
+   transferable and diverse.
+
+Suru brings a precise yet organic structure to the Ubuntu Touch interface. The sharp lines and varying levels of transparency evoke the edges and texture of paper.
+
+Design Tips for an App Icon
+---------------------------
+
+Descriptive design
+~~~~~~~~~~~~~~~~~~
+
+The icon you choose for your app should describe what your app is about, what it does, or what service it is there for in a recognizable way.
+
+.. figure:: /_static/images/humanguide/03.png
+
+Distinctive shapes
+~~~~~~~~~~~~~~~~~~
+
+Depending on the object you choose to show in the icon, on the perspective, lighting and other factors, it might be hard to recognize what you are trying to depict. A TV might look like a box, or a computer screen, a remote control like a phone, etc.
+
+.. figure:: /_static/images/humanguide/04.png
+
+Details
+~~~~~~~
+
+A detailed icon is very nice to look at as it shows quality and your eye for the small things. It might add confusion though, especially in smaller sizes or for people who have impaired eyesight. Show what's necessary, pay attention to detail, but don't overload the icon.
+
+.. figure:: /_static/images/humanguide/05.png
+
+Colours
+~~~~~~~
+
+Use distinctive colours, make your icon stand out a bit and give the shapes enough contrast to be visible.
+
+.. figure:: /_static/images/humanguide/06.png
+
+Gradients
+"""""""""
+
+If you were to add a gradient to your icon, only use colours that are slightly different from each other so that the change is not too dramatic. For example:
+
+.. figure:: /_static/images/humanguide/07.png
```

Comment:
Could you say exactly how wide the gradient is in or after this image, so people know exactly what you mean by "not too dramatic"?